### PR TITLE
[ES] Update `Ranking criteria`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Ranking_criteria/es.md [hunk 1]
```diff
@@ -113,9 +109,11 @@ Importante entender antes de usar:
...
-  - Nota: Si un corte no oficial contiene secciones coincidentes en el mismo orden y tiene aproximadamente la misma duración que el tamaño oficial para televisión, una versión corta o una versión para juegos, contará como corte oficial y se usará el marcador correspondiente. Las covers y remezclas no cuentan.
-- **Las versiones extendidas no oficiales de canciones deben añadir un marcador `(Extended Edit)` al final del título actual.** Si ya hay un marcador de duración en el título de la canción, `(Extended Edit)` lo sustituirá. Esto sirve para distinguir las versiones extendidas no oficiales de una canción de las versiones extendidas oficiales.
+- **Las versiones cortadas no oficiales de las canciones deben añadir un marcador `(Cut Ver.)` al final del título actual.** Si ya hay un marcador de duración en el título de la canción, `(Cut Ver.)` debe sustituirlo. Esto se hace para distinguir los cortes no oficiales de una canción de las versiones completas.
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Ranking_criteria/es.md:112]:
```suggestion
- **Las versiones cortadas no oficiales de las canciones deben añadir un marcador `(Cut Ver.)` al final del título actual.** Si ya hay un marcador de duración en el título de la canción, `(Cut Ver.)` debe reemplazarlo. Esto se hace para distinguir los cortes no oficiales de una canción de las versiones completas.
```

File: wiki/Ranking_criteria/es.md [hunk 2]
```diff
@@ -113,9 +109,11 @@ Importante entender antes de usar:
...
+  - **Las canciones que se cortan intencionalmente para recrear una versión oficial no se consideran versiones cortadas.** Esto solo se aplica si el audio es casi indistinguible de la versión oficial.
+  - **Las canciones que son un bucle completo de una pista en bucle no se consideran versiones cortadas.**
+- **Las versiones extendidas no oficiales de las canciones deben añadir un marcador `(Extended Edit)` al final del título actual.** Si ya hay un marcador de duración en el título de la pista, `(Extended Edit)` debe reemplazarlo. Esto sirve para distinguir las versiones extendidas no oficiales de una canción de las versiones extendidas oficiales.
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/Ranking_criteria/es.md:115]:
```suggestion
- **Las versiones extendidas no oficiales de las canciones deben añadir un marcador `(Extended Edit)` al final del título actual.** Si ya hay un marcador de duración en el título de la pista, `(Extended Edit)` debe reemplazarlo. Esto se hace para distinguir las versiones extendidas no oficiales de una canción de las versiones extendidas oficiales.
```

**Review by Unnombredeusuarioextremadamentelargo — CHANGES_REQUESTED** [review]

**Review by agatemosu — APPROVED** [review]

File: wiki/Ranking_criteria/es.md [hunk 3]
```diff
@@ -113,9 +109,11 @@ Importante entender antes de usar:
...
+- **Las versiones cortadas no oficiales de las canciones deben añadir un marcador `(Cut Ver.)` al final del título actual.** Si ya hay un marcador de duración en el título de la canción, `(Cut Ver.)` debe reemplazarlo. Esto se hace para distinguir los cortes no oficiales de una canción de las versiones completas.
+  - **Las canciones que se cortan intencionalmente para recrear una versión oficial no se consideran versiones cortadas.** Esto solo se aplica si el audio es casi indistinguible de la versión oficial.
+  - **Las canciones que son un bucle completo de una pista en bucle no se consideran versiones cortadas.**
+- **Las versiones extendidas no oficiales de las canciones deben añadir un marcador `(Extended Edit)` al final del título actual.** Si ya hay un marcador de duración en el título de la pista, `(Extended Edit)` debe reemplazarlo. Esto se hace para distinguir las versiones extendidas no oficiales de una canción de las versiones extendidas oficiales.
```
Comment (agatemosu) [inline wiki/Ranking_criteria/es.md:115]:
```suggestion
- **Las versiones extendidas no oficiales de las canciones deben añadir un marcador `(Extended Edit)` al final del título actual.** Si ya hay un marcador de duración en el título de la pista, `(Extended Edit)` debe sustituirlo. Esto se hace para distinguir las versiones extendidas no oficiales de una canción de las versiones extendidas oficiales.
```

File: wiki/Ranking_criteria/es.md [hunk 4]
```diff
@@ -113,9 +109,11 @@ Importante entender antes de usar:
...
-  - Nota: Si un corte no oficial contiene secciones coincidentes en el mismo orden y tiene aproximadamente la misma duración que el tamaño oficial para televisión, una versión corta o una versión para juegos, contará como corte oficial y se usará el marcador correspondiente. Las covers y remezclas no cuentan.
-- **Las versiones extendidas no oficiales de canciones deben añadir un marcador `(Extended Edit)` al final del título actual.** Si ya hay un marcador de duración en el título de la canción, `(Extended Edit)` lo sustituirá. Esto sirve para distinguir las versiones extendidas no oficiales de una canción de las versiones extendidas oficiales.
+- **Las versiones cortadas no oficiales de las canciones deben añadir un marcador `(Cut Ver.)` al final del título actual.** Si ya hay un marcador de duración en el título de la canción, `(Cut Ver.)` debe reemplazarlo. Esto se hace para distinguir los cortes no oficiales de una canción de las versiones completas.
```
Comment (agatemosu) [inline wiki/Ranking_criteria/es.md:112]:
```suggestion
- **Las versiones cortadas no oficiales de las canciones deben añadir un marcador `(Cut Ver.)` al final del título actual.** Si ya hay un marcador de duración en el título de la canción, `(Cut Ver.)` debe sustituirlo. Esto se hace para distinguir los cortes no oficiales de una canción de las versiones completas.
```

**Review by agatemosu — COMMENTED** [review]
podrías cambiar la línea 20 y 214 también

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**